src/index.js

```diff
@@ -47,6 +61,14 @@ class LndEngine {
       throw new Error(`${symbol} is not a valid symbol for this engine.`)
     }
 
+    // Expose config publicly that we expect to be used by consumers
+    PUBLIC_CONFIG.forEach((configKey) => {
+      if (this.currencyConfig[configKey] == null) {
```

Comment:
OK so JavaScript is weird with this one because of the coercion (`undefined == null`), can we use `this.currencyConfig.hasOwnProperty(configKey)` instead. We can then have values of null or 0 or whatever for the config files.

Reply:
I assumed we probably used this line to have a positive instead of negative statement*

Reply:
Yeah I did this to catch `undefined` and `null`, but still allow `0` and other falsey values. The double equals was intentional. But I think that `hasOwnProperty` is cleaner, I'll do that.